gradio/blocks.py

```diff
@@ -242,7 +249,11 @@ def get_config_file(self):
             config["components"].append(
                 {
                     "id": _id,
-                    "type": block.__class__.__name__.lower(),
+                    "type": (
```

Comment:
This could be `block.get_block_name()` given the method that was just added to the `Blocks` class

Reply:
good catch!

Reply:
well I actually mainly created the function to use here, but it seems I forgot, thx for the careful review.